saleor/checkout/complete_checkout.py

```diff
@@ -353,7 +352,7 @@ def _create_lines_for_order(
         additional_filter_lookup=additional_warehouse_lookup,
         existing_lines=lines,
         replace=True,
-        check_reservations=check_reservations,
+        check_reservations=True,
```

Comment:
What about the second flow for creating an order. It looks like this change has impact also on `_create_order_from_checkout` -> `_create_order_lines_from_checkout_lines` -> `_create_lines_for_order`.  Is it expected that we check reservation for it? The new flow is one step checkout (we don't process a payment like in checkout_complete), so I am not sure if we need to force checking reservation.

Reply:
@korycins I don't understand the above. What do you mean by "force checking reservation?" Every bit of code that uses stock needs to respect reservations.